### DISCUSSION TITLE
make tooltip visible by default / add tooltip signifier

### DISCUSSION THREAD

**Discussion by JacobBeaudry** [airbnb/visx#1377] [Q&A]
Is there any way to setup tooltips (_e.g._ on line-series) so that the tooltip for one datum would be visible by default? 

I'm creating graphs programmatically, so find it's next-to-impossible to specify the datum for `labels` in such a way that they are always placed somewhere sensible, so tooltips seem like the best option for labelling, in my case; however, I'd like there to be some indication to the end-user that the tooltips are there (_v.i._ for example); is there any way to do this? 

_e.g._ 
- specify the "display-datum" at which the tooltip is displayed if the user is not mousing over the graph; 
- snap to tooltip corresponding to mouse position, when user is mousing over the graph; 
-  when user is no longer mousing over, either: 
    - remain at last location, or
    - return to "display-datum"

**Comment by williaster** [comment]
Hi @RaxAdaam 👋  are you using the high-level `@visx/xychart` package, or the low-level packages for charts?

**Reply by JacobBeaudry** [reply]
Hi @williaster! We're using the high-level package. Thanks for your time!
